docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,441 @@
+# Contribute to the Keptn documentation
+
+This document provides information about contributing to
+the [Keptn Lifecycle Toolkit documentation](https://lifecycle.keptn.sh/docs/),
+which is part of the [Keptn](https://keptn.sh) website.
+
+The Keptn Lifecycle Toolkit documentation is authored with
+[markdown](https://www.markdownguide.org/basic-syntax/)
+and rendered using the Hugo
+[Docsy](https://www.docsy.dev/) theme.
+
+We welcome and encourage contributions of all levels.
+You can make modifications using the GitHub editor;
+this works well for small modifications but,
+if you are making significant changes,
+you may find it better to fork and clone the repository
+and make changes using the text editor or IDE of your choice.
+You can also run the Docsy based website locally
+to check the rendered documentation
+and then push your changes to the repository as a pull request.
+
+If you need help getting started,
+feel free to ask for help on the `#help-contributing` or `#keptn-docs` channels on the [Keptn Slack](https://keptn.sh/community/#slack).
+We were all new to this once and are happy to help you!
+
+## Guidelines for contributing
+
+* Always fork the repository then clone that fork to your local system
```

Comment:
I think we should also mention that PRs in the repo follow the semantic commit scheme. Therefore doc PRs should start with "docs:"

Reply:
Good idea!  I added it to the information about creating the PR, somewhere around line 241.